index.js

```diff
@@ -80,15 +81,62 @@ async function annotatePackage(packageName, filePath, lineNumber) {
     }
 }
 
+// Fetch modified lines in the PR
+async function getModifiedLines(filePath) {
+    const token = core.getInput('github-token', { required: true });
+    const octokit = github.getOctokit(token);
+
+    const { context } = github;
+    const { owner, repo } = context.repo;
+    const pull_number = context.payload.pull_request.number;
```

Comment:
Do we have access to `context.payload.pull_request.changed_files` ?

Reply:
or use context.payload.pull_request.base.ref to diff the files directly otherwise the user needs to set up a token and perms

Reply:
`context.payload.pull_request.number` just gets the general information about the PR, we get updated files and updated lines from the comand below: `octokit.rest.pulls.listFiles`